pkg/apis/v1/ec2nodeclass_status.go

```diff
@@ -24,6 +24,7 @@ const (
 	ConditionTypeSecurityGroupsReady  = "SecurityGroupsReady"
 	ConditionTypeAMIsReady            = "AMIsReady"
 	ConditionTypeInstanceProfileReady = "InstanceProfileReady"
+	ConditionTypeTagsReady            = "TagsReady"
```

Comment:
I feel like we should take a page out of what we are doing for NodePools and do something like `ValidationSucceeded`. @rschalo has a similar proposal in his tag validation PR